### ISSUE TITLE
java.lang.NoSuchFieldError: Companion

### ISSUE THREAD

**Issue by bunkyoneko** [acejump/AceJump#432]
**Describe the bug**
It started to happen after upgrading Intellij IDEA(Community Edition) to version 2022.3.3 Build #IC-223.8836.41.
The error would cause AceJump quit and not function any more.

```
java.lang.NoSuchFieldError: Companion
	at org.acejump.action.AceAction.actionPerformed(AceAction.kt:34)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:327)
	at com.intellij.openapi.keymap.impl.ActionProcessor.performAction(ActionProcessor.java:47)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:584)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$doPerformActionInner$9(IdeKeyEventDispatcher.java:706)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:105)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:94)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$doPerformActionInner$10(IdeKeyEventDispatcher.java:706)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:350)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.doPerformActionInner(IdeKeyEventDispatcher.java:703)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:647)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:595)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.java:478)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:467)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:225)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.java:815)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:750)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$5(IdeEventQueue.java:437)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:436)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueue.performActivity(IdeEventQueue.java:615)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:434)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:838)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:480)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

**To Reproduce**
1. Use Intellij IDEA and open any file.
2. Use `command + ;` to trigger AceJump
3. Error will pop up

**Desktop (please complete the following information):**
 - OS: MacOS 13.2 (22D49), Apple M1 Pro
 - IDE: Intellij IDEA(Community Edition) 2022.3.3 Build #IC-223.8836.41
 - AceJump version: 3.8.12

**Comment by Jacob3075** [comment]
Facing same issue
OS: windows
IDE: Android Studio Giraffe | 2022.3.1 Beta 2
Build #AI-223.8836.35.2231.10023527, built on April 27, 2023

**Comment by breandan** [comment]
Can you try updating to 2023.1 and reporting back @asapha @Jacob3075 @bunkyoneko?

**Comment by asapha** [comment]
@breandan It does works on `2023.1+`.

Tested on:
- Android Studio Hedgehog | 2023.1.1 Canary 3
- IntelliJ IDEA 2023.1.1 (Ultimate Edition)

**Comment by bunkyoneko** [comment]
@breandan Thanks for the reminder.
Tested on IntelliJ IDEA 2023.1.1 (Community Edition), and it works!

**Comment by ffkjjj** [comment]
Facing the same issue
IDE: IntelliJ IDEA 2023.3.3
After downgrading to AceJump 3.8.11, it works fine.

**Comment by breandan** [comment]
@ffkjjj `2023.3` has not been released yet, the latest is `2023.2 EAP`. Maybe you mean `2023.1`?

**Comment by ffkjjj** [comment]
> @ffkjjj `2023.3` has not been released yet, the latest is `2023.2 EAP`. Maybe you mean `2023.1`?

Sorry, I was wrong, my IDEA version is `2022.3.3`

**Comment by songan-1** [comment]
Facing same issue
OS: OSX
IDE: idea 2022.3 
plugin version: 3.8.13
